tensorflow/lite/experimental/micro/kernels/cmsis-nn/scratch_buffer.cc

```diff
@@ -15,20 +15,21 @@ limitations under the License.
 
 #include "scratch_buffer.h"
 
-// todo: remove this function once context->AllocateTemporaryTensor() is
-// implemented.
-
-// This buffer is used by CMSIS-NN optimized operator implementations.
-// SCRATCH_BUFFER_BYTES bytes is chosen empirically. It needs to be large
-// enough to hold the biggest buffer needed by all CMSIS-NN operators in the
-// network.
 #define SCRATCH_BUFFER_BYTES 13000
 
-__attribute__((aligned(
-    4))) static int16_t cmsis_scratch_buffer[SCRATCH_BUFFER_BYTES / 2] = {0};
-
+// todo: remove this function once context->AllocateTemporaryTensor() is
+// implemented.
+// note: buffer must be 32-bit aligned for SIMD
 TfLiteStatus get_cmsis_scratch_buffer(TfLiteContext* context, int16_t** buf,
                                       int32_t buf_size_bytes) {
+  // This buffer is used by CMSIS-NN optimized operator implementations.
+  // SCRATCH_BUFFER_BYTES bytes is chosen empirically. It needs to be large
+  // enough to hold the biggest buffer needed by all CMSIS-NN operators in the
+  // network.
+
+  __attribute__((aligned(
+      4))) static int16_t cmsis_scratch_buffer[SCRATCH_BUFFER_BYTES / 2] = {0};
```

Comment:
@lamarrr there's another PR from you fixing this as well. Just a heads up to avoid merge conflicts...

Reply:
I'll close that. Thanks

Reply:
@freddan80, I've added the kernel fallbacks. Please do a review and give feedback  ;-)